frontend/src/test/scala/bloop/bsp/BspBaseSuite.scala

```diff
@@ -750,7 +750,12 @@ abstract class BspBaseSuite extends BaseSuite with BspClientTest {
       val closeTask = {
         lsClient.request(endpoints.Build.shutdown, bsp.Shutdown()).flatMap { _ =>
           Task.fromFuture(lsClient.notify(endpoints.Build.exit, bsp.Exit())).map { _ =>
-            socket.close()
+            try {
+              socket.close()
```

Comment:
Maybe I'm totally missing it, but how would it be closed already if there was still a connection until `closeTask` was called?

Reply:
hmm.... not sure exactly, but this seems only a clean up, so it shouldn't fail anything